src/main/java/hwjdi/entities/UserProperties.java

```diff
@@ -0,0 +1,29 @@
+package hwjdi.entities;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class UserProperties extends Properties{
+
+    protected static UserProperties properties;
+
+    public static UserProperties getUserProperties() {
+
+        if (properties == null) {
+
+            properties = new UserProperties();
+
+            try (InputStream input = new FileInputStream("src/main/resources/hwjdi/config.properties")) {
```

Comment:
Could you please tell me, what is the reason of two catch blocks here ?

Reply:
That's because of auto suggestions from Intellij idea, but since class FileNotFoundException extends IOException, it's just redundant. Removed.